src/utilities/bindHttpMethod.js

```diff
@@ -13,7 +13,8 @@ export default (originalMethod: Function, agent: AgentType, forceGlobalAgent: bo
     let options;
     let callback;
 
-    if (typeof args[0] === 'string') {
+    // the url parameter can be string or URL
```

Comment:
The comment is superfluous. That is literally what is expressed in the next line.

Reply:
It might be, but what it serves as a reminder, so that in the future this kind of careless bug should be avoided.